contrib/gorm.io/gorm.v1/example_test.go

```diff
@@ -41,6 +42,30 @@ func ExampleOpen() {
 	db.Where("name = ?", "jinzhu").First(&user)
 }
 
+func ExamplePlugin() {
```

Comment:
```suggestion
// ExampleNewTracePlugin illustrates how to trace gorm using the gorm.Plugin api. 
func ExampleNewTracePlugin() {
```

(so the example shows up with the `NewTracePlugin` documentation)